src/components/group/obs-group.component.tsx

```diff
@@ -11,42 +11,48 @@ export const ObsGroup: React.FC<FormFieldInputProps> = ({ field, ...restProps })
   const { t } = useTranslation();
   const { formFieldAdapters, formFields } = useFormProviderContext();
 
-  const content = useMemo(
```

Comment:
Assuming the issue is related to how form intents are handled in the `src/utils/forms-loader.ts`, is it necessary to make changes in this file?